sysutils/dmidecode/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=		dmidecode
+PLUGIN_VERSION=		1.0
```

Comment:
we want to start with a clean slate :)

PLUGIN_VERSION=0.1
PLUGIN_DEVEL=yes

Reply:
done